src/UI/Implementation/Component/Listing/DescriptiveList/DescriptiveList.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+/* Copyright (c) 2016 Timon Amstutz <timon.amstutz@ilub.unibe.ch> Extended GPL, see docs/LICENSE */
+
+namespace ILIAS\UI\Implementation\Component\Listing\DescriptiveList;
+
+use ILIAS\UI\Component as C;
+
+/**
+ * Class DescriptiveList
+ * @package ILIAS\UI\Implementation\Component\Listing\DescriptiveList
+ */
+class DescriptiveList implements C\Listing\DescriptiveList {
+
+	/**
+	 * @var	array
+	 */
+	private  $items;
+
+	/**
+	 * @inheritdoc
+	 */
+	public function __construct(array $items) {
```

Comment:
This results from not having a text component. If we would have a text component, I could check for UI->component or a specific type of UI component. Since we to not have a text component anymore, I could path string or a component (you right, the type hinting needs to be adapted). How would a check for this?